back_office/add_products/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class AddProductsConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'back_office.add_products'
```

Comment:
App should be called just products, as there will be A LOT more functionality in here than just add products, we will delete, update, edit and do all the product related things here.

I suggest delete this folder and create new app named products.

Reply:
wil change to products then.